src/components/ButtonWalletPredicate/ButtonWalletPredicate.tsx

```diff
@@ -0,0 +1,44 @@
+import type { ButtonProps } from '@chakra-ui/react'
+import { Button } from '@chakra-ui/react'
+import type { InterpolationOptions } from 'node-polyglot'
+import { useCallback } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { useWallet } from 'hooks/useWallet/useWallet'
+
+type ButtonWalletPredicateProps = {
+  isValidWallet: boolean
+  invalidWalletTranslation?: string | [string, InterpolationOptions]
+  onWalletNotConnectedClick?: () => void
+} & ButtonProps
+
+export const ButtonWalletPredicate = ({
+  isValidWallet,
+  invalidWalletTranslation = 'common.connectWallet',
```

Comment:
q: This isn't passed down currently, can we see ay scenario when we want to pass any other copy, or can we remove this prop?

Reply:
fml

Reply:
Fixed in [1b98335](https://github.com/shapeshift/web/pull/7270/commits/1b98335766e29210cd5969426bc7506fc19efea8)